commit 2dd939575039b437c5e2e9e26a2b4824ce232849
Author: Nicolas Morey <nmorey@suse.com>
Date:   Mon Sep 14 17:10:01 2026 +0200

    Revert "prov/verbs: Add peer mem support for Broadcom Devices"
    
    This reverts commit d0f4f2defbd19e4ac42fc60bae6bc14f13e4612a.

diff --git prov/verbs/src/verbs_info.c prov/verbs/src/verbs_info.c
index 2c947800cacc..3751b7cc9ba2 100644
--- prov/verbs/src/verbs_info.c
+++ prov/verbs/src/verbs_info.c
@@ -801,7 +801,6 @@ static bool vrb_hmem_supported(struct ibv_context *ctx)
 		return false;
 
 	if (vrb_gl_data.peer_mem_support && (strstr(dev_name, "mlx") ||
-					     strstr(dev_name, "bnxt_re") ||
 					     strstr(dev_name, "roce")))
 		return true;
 
